README.md

```diff
@@ -1 +1,66 @@
-# react-todo-app
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+> Hello! I am a software developer! I can help you build a product, feature or website. Tak a look of my works.
```

Comment:
[Optional] Kindly suggest modifying the project description. Or just removing this part. Looks like is part of your old project template.

Reply:
@wandergithub Done.